project/templates/configs/pyramid.ini

```diff
@@ -19,33 +5,21 @@ pyramid.reload_templates = false
 pyramid.debug_authorization = false
 pyramid.debug_notfound = false
 pyramid.debug_routematch = false
+
 pyramid.default_locale_name = {{default_language}}
 
-available_languages = {{available_languages|safe}}
+git.path = {{ucd_host}}/repos/{{repo_name}}.json
 
-git.path = {{repo_path}}
-git.content_repo_url = {{git_repo_uri}}
+thumbor.security_key = {{thumbor_security_key}}
 
 es.host = {{es_host}}
-es.index_prefix = unicore_frontend_{{app_type}}_{{country}}
```

Comment:
`es.index_prefix` is still used. it should equal `{{repo_name}}`. It's so that websites can still work with their local repo setup.